deps/rabbitmq_aws/src/rabbitmq_aws.erl

```diff
@@ -543,10 +543,22 @@ ensure_credentials_valid() ->
 %% @end
 api_get_request(Service, Path) ->
   rabbit_log:debug("Invoking AWS request {Service: ~p; Path: ~p}...", [Service, Path]),
+  api_get_request_with_retries(Service, Path, ?MAX_RETRIES).
+
+-spec api_get_request_with_retries(string(), path(), number()) -> result().
+%% @doc Invoke an API call to an AWS service with retries.
+%% @end
+api_get_request_with_retries(Service, Path, Retries) ->
   ensure_credentials_valid(),
   case get(Service, Path) of
     {ok, {_Headers, Payload}} -> rabbit_log:debug("AWS request: ~s~nResponse: ~p", [Path, Payload]),
-                                 {ok, Payload};
+    {ok, Payload};
     {error, {credentials, _}} -> {error, credentials};
-    {error, Message, _}       -> {error, Message}
+    {error, Message, _}       ->
+      case Retries of
+        0 -> {error, Message};
+        _ -> rabbit_log:warning("Failed to make AWS request: ~p. Will retry (Retries left: ~p)...", [Message, Retries]),
+          timer:sleep(?LINEAR_BACK_OFF_MILLIS),
+          get(Service, Path, Retries - 1)
```

Comment:
this should be  `api_get_request_with_retries`? 😜

Reply:
Hm?? How does my test passed... Let me try again.